### DISCUSSION TITLE
Antd 4.11.2 Tree

### DISCUSSION THREAD

**Discussion by dbaranov7658** [ant-design/ant-design#29732] [Q&A]
Can someone please provide an example of using function scrollTo an element of a tree. 
"treeRef.current.scrollTo({ key: "0-0-0-5-4" });"

**Comment by tsteenbe** [comment]
See for an example https://github.com/react-component/tree/blob/a003237bb774dd0387450271409f29427654018f/docs/examples/animation.jsx
